src/hashset.c

```diff
@@ -1291,50 +1291,51 @@ size_t hashsetScan(hashset *t, size_t cursor, hashsetScanFunction fn, void *priv
             /* Advance cursor. */
             cursor = nextCursor(cursor, mask);
         } else {
-            /* Let table0 be the the smaller table and table1 the bigger one. */
-            int table0, table1;
-            if (t->bucketExp[0] <= t->bucketExp[1]) {
-                table0 = 0;
-                table1 = 1;
+            int growing = t->bucketExp[0] <= t->bucketExp[1];
+            int table_small, table_large;
+            if (growing) {
```

Comment:
The `growing` variable is only used in one place. It doesn't add any clarity IMO.

```suggestion
            int table_small, table_large;
            if (t->bucketExp[0] <= t->bucketExp[1]) {
```